src/app/dashboard/investments/page.tsx

```diff
@@ -15,11 +15,11 @@ import Money from '@/book/Money';
 import * as API from '@/hooks/api';
 
 export default function InvestmentsPage(): JSX.Element {
-  const { data: investments, isLoading } = API.useInvestments();
+  const { data: investments, isPending } = API.useInvestments();
   const { data: currency } = API.useMainCurrency();
   const mainCurrency = currency?.mnemonic || 'EUR';
 
-  if (isLoading) {
+  if (isPending) {
```

Comment:
`isPending` is the recommended approach as we don't want to show `loading` when we fetch data from other keys